README.md

```diff
@@ -35,6 +35,29 @@ export KUBECONFIG=.gcp/admin.kubeconfig
 kubectl api-resources
 ```
 
+## Batteries
+
+Example server contains a simple implementation of batteries that can be used to extend the gcp API.
+
+Batteries:
+- `leases` - a Kubernetes lease resources from `coordination.k8s.io`
+- `authentication` - Kubernetes native authentication using `authentication.k8s.io`
+- `authorization` - Kubernetes native authorization using `authorization.k8s.io`
+- `admission` - Kubernetes native admission using `admissionregistration.k8s.io`
+- `flowcontrol` - Kubernetes native flow control using `flowcontrol.apiserver.k8s.io`
+
+
+When starting server without any flags, in-memory storage will be used and batteries will be disabled by default.
```

Comment:
what does in-memory storage mean? You mean you wire through no-op informers I assume?

Reply:
in-memory etcd - our usual standalone development mode.

Reply:
ack